operator/main.go

```diff
@@ -148,6 +159,79 @@ func runOperator(cmd *cobra.Command) {
 	}
 	close(k8sInitDone)
 
+	k8sversion.Update(k8s.Client(), option.Config)
+	if !k8sversion.Capabilities().MinimalVersionMet {
+		log.Fatalf("Minimal kubernetes version not met: %s < %s",
+			k8sversion.Version(), k8sversion.MinimalVersionConstraint)
+	}
+
+	// Get hostname for identity name of the lease lock holder.
+	// We identify the leader of the operator cluster using hostname.
+	hostID, err := os.Hostname()
+	if err != nil {
+		log.WithError(err).Fatal("failed to get hostname")
+	}
+	hostID = rand.RandomStringWithPrefix(hostID+"-", 8)
+	log.Infof("using host ID %s for operator POD", hostID)
+
+	ns := option.K8sNamespaceName
```

Comment:
will it be `ns := viper.GetString(option.K8sNamespaceName)` instead ?

Reply:
@sayboras no, the `option.K8sNamespaceName` is set in https://github.com/cilium/cilium/blob/edff374340cb55318ea3fffba13322f9152cdaf6/pkg/option/config.go#L2535